### DISCUSSION TITLE
How do I identify FE (npm) dependencies from BE dependencies (so I can label them)?

### DISCUSSION THREAD

**Discussion by slonim-greg** [renovatebot/renovate#23697] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.27.1

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Wanted end result.

PRs/MRs related to front-end libraries (npm) are labeled. 

In order for me to do so, I need to build a package regex. How would I go about identifying differences in MRs/PRs that deal with npm deps vs back end deps (in my case, Nuget packages)?

### What you tried so far.

Have not tried anything yet.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
package rules with `matchManagers` ?

**Reply by slonim-greg** [reply]
Thanks, Michael, I can try that.
